test/dotnet-cache.Tests/GivenDotnetCachesAndPublishesProjects.cs

```diff
@@ -88,7 +89,7 @@ public void AppFailsDueToMissingCache()
                 .CreateInstance()
                 .WithSourceFiles()
                 .Root.FullName;
-            var profileProject = Path.Combine(profileProjectPath, $"{profileProjectName}.xml");
+            var profileProject = Path.Combine(profileProjectPath, "NewtonsoftFilterProfile.xml");
```

Comment:
Why the inconsistency with the naming? One is profilefilter, the other is filterprofile? Can we stick with one?